packages/atomic/src/components/search/result-template-components/atomic-result-date/atomic-result-date.tsx

```diff
@@ -36,6 +41,13 @@ export class AtomicResultDate implements InitializableComponent {
    */
   @Prop({reflect: true}) format = 'D/M/YYYY';
 
+  /**
+   * Specify that date should display as calendar time: https://day.js.org/docs/en/plugin/calendar.
+   *
+   * To modify the calendar time string, use the localization feature: https://docs.coveo.com/en/atomic/latest/usage/atomic-localization/
+   */
+  @Prop({reflect: true}) calendarTime?: boolean;
```

Comment:
Since this is a boolean, I would make the property name start with a verb, like `useCalendarTime`.

Reply:
Also, instinctively I expect "calendar time" to mean a format like "2020-01-02". Perhaps something like "relative time"/`useRelativeTime` would be more intuitive.

Reply:
Calendar time is the verbiage they use both for dayjs (https://day.js.org/docs/en/plugin/calendar) and momentjs (https://momentjs.com/docs/#/displaying/calendar-time/)

Reply:
I'm in favor of `relativeTime` because: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat

Reply:
Modified to `relativeTime`